packages/ui/src/theme/tokens/components/toggleButton.js

```diff
@@ -0,0 +1,74 @@
+module.exports = {
+  borderColor: { value: '{colors.border.secondary.value}' },
+  color: { value: 'rgba(0, 0, 0, 0.54)' },
+  _hover: {
+    backgroundColor: { value: 'rgba(0, 0, 0, 0.04)' },
+  },
+  _focus: {
+    borderColor: { value: '{colors.focus.value}' },
+    color: { value: 'rgba(0, 0, 0, 0.54)' },
+  },
+  _active: {
+    backgroundColor: { value: '{colors.transparent.value}' },
+  },
+  _disabled: {
+    backgroundColor: { value: '{colors.transparent.value}' },
+    borderColor: { value: '{colors.border.secondary.value}' },
+    color: { value: '{colors.font.disabled.value}' },
+  },
+  pressed: {
+    color: { value: 'rgba(0, 0, 0, 0.87)' },
+    backgroundColor: { value: 'rgba(0, 0, 0, 0.08)' },
+    _hover: {
+      backgroundColor: { value: 'rgba(0, 0, 0, 0.12)' },
+    },
+  },
+  primary: {
+    backgroundColor: { value: '{colors.transparent.value}' },
+    borderWidth: { value: '{borderWidths.small.value}' },
+    _focus: {
+      borderColor: { value: '{colors.focus.value}' },
+      backgroundColor: { value: '{colors.transparent.value}' },
+      boxShadow: { value: '0 0 0 1px {colors.focus.value}' },
+      color: { value: 'rgba(0, 0, 0, 0.54)' },
+    },
+    _hover: {
+      backgroundColor: { value: 'rgba(0, 0, 0, 0.04)' },
+      color: { value: 'rgba(0, 0, 0, 0.54)' },
+    },
+    _disabled: {
+      backgroundColor: { value: '{colors.background.tertiary.value}' },
+      color: { value: '{colors.font.disabled.value}' },
+    },
+    pressed: {
+      backgroundColor: { value: 'rgba(0, 128, 128, 0.08)' },
```

Comment:
Extract color to `colors.js` or use one of our default colors? (same applies to other rgba colors here)

Reply:
The default colors don't give me the granularity to set the color for each state, while I can achieve that by fine tuning the opacity of the same color using either `rgba` or` hsla`.

Do you have any ideas about how to extract that in to `color.js`? It seems we never do that for a specific primitive